beanstalk_env/README.md

```diff
@@ -0,0 +1,29 @@
+# beanstalk_env
```

Comment:
What is the relationship between a EB environment and an EB app? I don't see them related in the examples here? Is it based on the name matching?

Reply:
Oops, yeah the example is missing. I'll add it.

Conceptually, EB environments exist within an EB app, although the app doesn't provide anything except grouping multiple environments together. In this case they're linked together by the [app_name variable](https://github.com/openoakland/terraform-modules/pull/2/files#diff-904b2de3bbbe52eb478090768ca41ba7R53).

Reply:
FYI I don't really know EB that well.. so I'm sure this is going to change again. One thing I recently realized is that if you want workers, you specify a worker environment, which instead of a load balancer and auto-scaling group, gives you an SQS queue and auto-scaling group. So... `beanstalk_env` should really be `beanstal_web_env` :man_shrugging:

Reply:
Renamed!